cmd/txt-suite/cmd.go

```diff
@@ -23,12 +26,31 @@ func stayAlive() bool {
 	return *keeprunning
 }
 
+func showVersion() {
```

Comment:
just wrong

- relative path
- a secondary file
- no prefix before Version makes it harder to parse it using scripting languages
- arbitrary text before Version makes it harder to parse it using scripting languages